card/lib/card/spec_helper.rb

```diff
@@ -76,4 +76,33 @@ def render_card_with_args view, card_args={}, format_args={}, view_args={}
   def users
     SharedData::USERS.sort
   end
+
+  def in_phase opts, &event_block
```

Comment:
Assignment Branch Condition size for in_phase is too high. [20.71/15]<br>Method has too many lines. [25/15]